invenio_rdm_records/cli.py

```diff
@@ -48,7 +48,7 @@ def create_fake_record():
             "subtype": "photo"
         },
         "identifiers": [{
-            "identifier": "10.5281/zenodo.9999999",
+            "identifier": "99.9999/rdm.9999999",
```

Comment:
DOI's always start with ``10.`` so if at any point we need to validate them, it's better to have it start with ``10.``

Reply:
I was just trying not to collide with anything non-existent :) but having rdm there already ensures that. Chaging.